src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -521,6 +561,25 @@ public void SolutionBindingUpdated_WhenClearBoundProject_NotRaised()
                 eventCounter.SolutionBindingChangedCount.Should().Be(0);
             }
         }
+        
+        [TestMethod]
+        public void HandleBindingChange_WhenClearBoundProject_NotRaised()
+        {
```

Comment:
This test seems to be a copy-past of SolutionBindingUpdated_WhenClearBoundProject_NotRaised.
Can we maybe extract the whole content in a method and provide the act as a parameter to the method?